src/services/WolfDen.Domain/Entity/Status.cs

```diff
@@ -0,0 +1,29 @@
+﻿
+
+namespace WolfDen.Domain.Entity
+{
+    public class Status

```

Comment:
What does this table indicates

Reply:
To store the status of each day like present,absent..